enigma-core/app/src/networking/ipc_listener.rs

```diff
@@ -314,6 +318,8 @@ pub(self) mod handling {
         let msg = response.response.from_hex()?;
         km_u::ptt_res(eid, &msg)?;
         let res = km_u::ptt_build_state(db, eid)?;
+        // update state status
```

Comment:
This comment is redundant, since it repeats the function name. I would add here a more informative comment like you did  in case of `db.update_state_status(false)`